EKF/common.h

```diff
@@ -133,6 +133,7 @@ struct baroSample {
 struct rangeSample {
 	float       rng;	///< range (distance to ground) measurement (m)
 	uint64_t    time_us;	///< timestamp of the measurement (uSec)
+	uint8_t	    quality; ///< quality indicator between 0 and 255
```

Comment:
The signal quality metric in the PX4 distance_sensor topic is cast as a int8 with a range between 0 and 100 when quality is known and -1 when it is not. We should keep that mapping.

Reply:
good point